### PR TITLE
Add Travis Build

### PR DESCRIPTION
**Pull request by dbellettini**
@onebip-admin will have to enable it then

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# vim: set tabstop=2 shiftwidth=2 expandtab:
+language: php
+
+php:
+  - 5.4
```
Comment (giorgiosironi) [inline .travis.yml:5]: are we actually supporting all of these versions?

Comment (dbellettini) [inline .travis.yml:5]: You're right, I would keep only 5.4 and 5.6 and then keep only the latter after our upgrade

**Comment by giorgiosironi** [comment]
Still showing too many versions

**Comment by giorgiosironi** [comment]
I can't enable it because someone changed the password of onebip-admin

**Pull request merged**